assets/js/script.js

```diff
@@ -95,25 +95,43 @@ $(document).on('click', '.js-bag-link', function(evt) {
 $(document).on('click', '.js-remove-letter', function(evt) {
   var letters = state.letters.split('')
   letters.splice(state.activeLetter, 1)
-
   setState({
     letters: letters.join(''),
     editing: false,
   })
 })
 
-
 // edit individual letter
 $(document).on('click', '.js-letter', function(evt) {
   evt.stopPropagation()
 
   setState({
-    editing:      true,
-    editingBag:   false,
+    editing: true,
+    editingBag: false,
     activeLetter: $(this).data('index'),
   })
 })
 
+$(document).on('click', '.js-tab', function(evt) {
+  evt.stopPropagation()
+
+  if ($(this).data('index') > -1) {
+    setState({
+      editing: true,
+      editingBag: false,
+      activeLetter: $(this).data('index'),
+    })
+  } else {
+    setState({
+      editing: false,
+      editingBag: true,
+      activeLetter: $(this).data('index'),
```

Comment:
I'm glad that we're setting `activeLetter` to `-1` now, it's cleaner, but have you checked to make sure all of the code that relies on `activeLetter` works correctly when it's set to -1?